src/Template/Element/Form/relations.twig

```diff
@@ -112,10 +110,15 @@
                     :hide-relations="hideRelations">
                     <div class="relationships-view">
                         <transition name="fade">
-                            <div class="modal-view" v-show="isVisible" @keyup.esc="handleKeyboard($event)">
+                            <div class="modal-view" v-show="isVisible"
+                                {# @keyup.esc.prevent.stop="handleKeyboard($event)"> #}
+
+                                v-hotkey="keyEvents">
+
                                 <div class="modal-box">
                                     <div class="modal-header">
-                                        <button class="close" @click.prevent="hideRelationshipModal()">&times;</button>
+                                        <button class="close"
+                                            @click.prevent="hideRelationshipModal()">&times;</button>
```

Comment:
what is `&times;`? Button labels should be i18n IMHO, i.e.: `{{ __(' ... whatever ...') }}`

Reply:
è una x